src/client.rs

```diff
@@ -84,6 +86,7 @@ impl ClientBuilder {
             private_ips: enable_exclude(self.exclude_private_ips.unwrap_or_default()),
             link_local_ips: enable_exclude(self.exclude_link_local_ips.unwrap_or_default()),
             loopback_ips: enable_exclude(self.exclude_loopback_ips.unwrap_or_default()),
+            mail: enable_exclude(self.exclude_mail.unwrap_or_default()),
```

Comment:
@mre
Was it actually intended that `-E, --exclude-all-private` excludes Mails as well (via above line)?
Now here: https://github.com/lycheeverse/lychee/blob/8fd34a7/lychee-lib/src/client.rs#L122

Just recognised it and didn't expect it, also as help/readme state different.

Reply:
Whoops, no that was not intended. Nice catch!

Reply:
See https://github.com/lycheeverse/lychee/pull/316